cookbooks/aws-parallelcluster-slurm/recipes/install_jwt.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: install_jwt
+#
+# Copyright:: Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+libjwt_version = 'v1.12.0'
+libjwt_url = 'https://github.com/benmcollins/libjwt.git'
+
+jwt_build_deps = value_for_platform(
+  'ubuntu' => {
+    'default' => 'libjansson-dev',
+  },
+  'default' => 'jansson-devel'
+)
```

Comment:
I'm wondering if we are good with downloading this from os package repo. Shall we check what version is available in the various repos and if this package is available on ARM? For example libjson is not available on arm. The alternative would be to build this as well.

Reply:
Here's what I found available on various clusters:
From alinux2 cluster on arm: `jansson-devel.aarch64 0:2.10-1.amzn2.0.2`
From ubuntu20_04 on arm (after doing `sudo apt update`): `libjansson-dev_2.12-1build1_arm64`
From ubuntu18_04 on arm (after doing `sudo apt update`): `libjansson-dev_2.11-1_arm64`